README.org

```diff
@@ -1652,10 +1652,9 @@ Files names can contain any character that the file system
 permits. Denote imposes a few additional restrictions:
 
 + The tokens "==", =__= and =--= are interpreted by Denote and should
-  appear only once. =_= is the separator for keywords.
+  appear only once.
```

Comment:
Seems like "should appear only once" is outdated wording?

Reply:
I did not want to overwhelm users with users in the manual even though we are technically more permissive.